cardano-api/src/Cardano/Api/LedgerState.hs

```diff
@@ -928,13 +975,11 @@ tickThenApply
     -> Shelley.LedgerState
         (HFC.HardForkBlock
             (Consensus.CardanoEras Shelley.StandardCrypto))
-    -> Either Text (Shelley.LedgerState
-        (HFC.HardForkBlock
-            (Consensus.CardanoEras Shelley.StandardCrypto)))
+    -> Either Text LedgerStateEvents
 tickThenApply cfg block lsb
-  = either (Left . Text.pack . show) Right
-  $ runExcept
-  $ Ledger.tickThenApply cfg block lsb
```

Comment:
I just pushed a refined interface to PR input-output-hk/ouroboros-network#3292

I think this should now just be `... $ runExcept $ Ledger.tickThenApplyLedgerResult cfg block lsb`, no need for `runLedgerT`.